src/ImageDeleteModal.jsx

```diff
@@ -81,11 +75,20 @@ export const ImageDeleteModal = ({ imageWillDelete, onAddNotification }) => {
         }
     };
 
+    const imageName = repoTags[0]?.split(":")[0].split("/").at(-1) ?? "";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1451-20231115-110156-11caf125-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)